Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -67,7 +67,7 @@ public ValidationResult GetValidationResult((Contest?, int?, string, bool, bool)
         {
             if (participant.ParticipationEndTime != null)
             {
-                return DateTime.Now >= participant.ParticipationEndTime;
+                return DateTime.Now <= participant.ParticipationEndTime;
```

Comment:
If the contest is not expired, i.e. DateTime.Now <= participant.ParticipationEndTime, then this method named IsContestExpired will return true.